src/mechanisms/scram/parser.rs

```diff
@@ -46,6 +46,7 @@ impl SaslEscapeState {
     }
 }
 
+#[allow(clippy::copy_iterator)]
```

Comment:
This should be addressed

Reply:
I've opened https://github.com/dequbed/rsasl/issues/20 for this